Zeplin.sketchplugin/Contents/Sketch/main.cocoascript

```diff
@@ -1,17 +1,21 @@
 @import "utils.cocoascript";
 
 var onRun = function (context) {
-    var temporaryPath = temporaryPathForDesignFile(context);
-    if (!temporaryPath) {
-        return;
-    }
-    
     var artboards = [context valueForKeyPath:@"selection.@distinctUnionOfObjects.parentArtboard"];
     if (![artboards count]) {
         [NSApp displayDialog:@"Please select the artboards you want to export to Zeplin.\n\n☝️ Selecting a layer inside the artboard should be enough." withTitle:@"No artboard selected"];
         return;
     }
 
+    if (!isSelectionExportable(artboards)) {
+        return;
+    }
+
+    var temporaryPath = temporaryPathForDesignFile(context);
+    if (!temporaryPath) {
+        return;
+    }
```

Comment:
Do we want to warn user about the frame size before checking the temp path? Just wanted to be sure the check order is changed willingly.

Reply:
Yeap, this is even better since no need to create a `temporaryPath` if there is an oversized artboard in the selection.